src/app/GroupCreator.jsx

```diff
@@ -109,24 +119,17 @@ const styles = {
   },
   userListContainer: {
     marginTop: 5,
-    height: '100%',
-    minWidth: 400,
-    '@media (max-width: 1000px)': {
-      marginLeft: -20
-    },
-    display: 'flex',
-    width: '100%'
+    height: 900,
```

Comment:
I played with that but had really big problems with react-infinite. It was a nightmare :p happy to get some help with it.